sqlmesh/dbt/seed.py

```diff
@@ -31,10 +32,18 @@ class SeedConfig(BaseModelConfig):
     """
 
     delimiter: str = ","
+    column_types: t.Optional[t.Dict[str, str]] = None
+    quote_columns: t.Optional[bool] = False
 
     def to_sqlmesh(self, context: DbtContext) -> Model:
         """Converts the dbt seed into a SQLMesh model."""
         seed_path = self.path.absolute().as_posix()
+
+        for name, data_type in (self.column_types or {}).items():
+            column = self.columns.setdefault(name, ColumnConfig(name=name))
```

Comment:
Can we somehow avoid mutating the SeedConfig object and copy the `columns` instead? Otherwise, it appears that `to_sqlmesh` now has a side-effect which is not expected

Reply:
Yes it might not be the perfect solution, but to avoid calling `column_types_to_sqlmesh` twice I chose to copy the columns and after doing the translation in `sqlmesh_model_kwargs` (which uses `self.columns`) reset them back to the original

Reply:
sure refactored this to accept this optional dictionary